api/sendMail.ts

```diff
@@ -0,0 +1,53 @@
+import axios, { AxiosError } from "axios";
```

Comment:
Also your directory name could be different because you have another api folder in your next app, and also this file container sendMail class so maybe you could name directory like 'services'.